third_party/move/move-core/types/src/vm_status.rs

```diff
@@ -162,6 +168,15 @@ impl VMStatus {
         self.status_code().status_type()
     }
 
+    pub fn is_aggregator_error(&self) -> bool {
+        if let VMStatus::MoveAbort(_, code) = self {
```

Comment:
I think something like this can work?

`matches!(self, VMStatus::MoveAbort(_, code) if *code == EADD_OVERFLOW || *code == ESUB_UNDERFLOW)`

Reply:
That was what I tried first. But the compiler didn't recognize `code` in `*code == EADD_OVERFLOW`.

Reply:
I think `*code` should be binding to `code`?

Reply:
Done.